website/react-magma-docs/src/pages/data-visualization-intro/introduction.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Getting Started
+order: 1
+---
+
+import { LeadParagraph } from '../../components/LeadParagraph';
+import { Flex, FlexBehavior, FlexAlignItems, magma } from 'react-magma-dom';
+
+<PageContent componentName="getting_started" type="data_visualization_intro">
+
+<LeadParagraph>
+  Magma charts enable you to convey precise and compelling narratives around
+  data through visually appealing and accessible visualizations. 
+</LeadParagraph>
+
+Magma Charts is based on [Carbon Charts from IBM](https://charts.carbondesignsystem.com/introduction).
+
+Get started below with implementation instructions for developers. You can also learn more about the [chart types](../../data-visualization/chart-types) we provide, as well as [view examples](../../data-visualization/chart-demos).
+
+export const MagmaChartsSiteLink = () => (
+  <Link
+    to="https://storybook-preview-1544--upbeat-sinoussi-f675aa.netlify.app/?path=/story/carbonchart"
+    target="_blank"
+  >
+    <Flex
+      behavior={FlexBehavior.container}
+      alignItems={FlexAlignItems.center}
+      style={{
+        backgroundColor: magma.colors.neutral200,
+        width: 'fit-content',
+        gap: magma.spaceScale.spacing03,
+        padding: `${magma.spaceScale.spacing04} ${magma.spaceScale.spacing05}`,
+        borderRadius: magma.spaceScale.spacing03,
+      }}
+    >
+      <svg
```

Comment:
Since this Storybook svg gets used a few times, we should save it in its own file and then just reuse it.